account_netting/__openerp__.py

```diff
@@ -1,19 +1,22 @@
 # -*- coding: utf-8 -*-
 # (c) 2015 Pedro M. Baeza
+# (c) 2016 Noviat nv/sa (www.noviat.com)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
     'name': 'Account netting',
-    'version': '8.0.1.0.0',
+    'version': '8.0.2.0.0',
     'summary': "Compensate AR/AP accounts from the same partner",
     'category': 'Accounting & Finance',
     'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
```

Comment:
Can you please take the occasion to switch the authorship to my new company `Tecnativa`?